### ISSUE TITLE
PCRE engine in PHP doesn't work

### ISSUE THREAD

**Issue by itskenny0** [tbodt/ish#183]
This might be a fun one - the regular expression engine in PHP doesn't work and just doesn't match anything, no errors given.

How to reproduce:

- apk add php
- create test.php:
```
<?php
$rex = "/posts/i";
$in = "/posts/103331";

preg_match($rex, $in, $out);
print_r($out);
```

Output on Ubuntu 18.04 php7.2:
```
root@ooboontoo:~# php test.php
Array
(
    [0] => posts
)
```

Output on iSH php7.2:
```
kyPhoneXS:~# php test.php
Array
(
)
```

**Comment by tbodt** [comment]
This is an aarch64-only bug.

**Comment by tbodt** [comment]
Fixed in 5006a3c
